Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -911,14 +910,13 @@ getPaymentHistoryEntityDetails merchantShortId driverId invoiceId = do
 ---------------------------------------------------------------------
 clearOnRideStuckDrivers :: ShortId DM.Merchant -> Maybe Int -> Flow Common.ClearOnRideStuckDriversRes
 clearOnRideStuckDrivers merchantShortId dbSyncTime = do
-  merchant <- findMerchantByShortId merchantShortId
+  _merchant <- findMerchantByShortId merchantShortId
   now <- getCurrentTime
   let dbSyncInterVal = addUTCTime (fromIntegral (- fromMaybe 1 dbSyncTime) * 60) now
   driverInfos <- B.runInReplica $ QPerson.getOnRideStuckDriverIds dbSyncInterVal
   driverIds <-
     mapM
       ( \driverInf -> do
-          void $ DLoc.updateOnRide merchant.id driverInf.driverId False
```

Comment:
@chakri642 Here we should handle by calling our rideDetails API with status CANCELLED

Reply:
Unless if it is already getting handled by the Ride Cancel flow where you have called LTS rideDetails API